linode/manager
# upcoming: [M3-7607] Placement Groups - Queries, Types, Validation, Factories and Mock Data

## Description 📝
This PR adds Types, Queries, Validation, Factories and MSW data in order to get mock based development started for the upcoming Placement Group feature.

## Changes  🔄
- See above description

## How to test 🧪

### Verification steps 
(How to verify changes)
- verify code and naming conventions
- test hooks with MSW turned on to confirm data flows

## As an Author I have considered 🤔

*Check all that apply*
